finetune/base_models/textcnn/model.py

```diff
@@ -0,0 +1,22 @@
+from finetune.base_models import SourceModel
+from finetune.base_models.gpt.encoder import GPTEncoder
+from finetune.base_models.textcnn.featurizer import textcnn_featurizer
+# from finetune.config import GridSearchable
+
+class TextCNNModel(SourceModel):
+
+    encoder = GPTEncoder
+    featurizer = textcnn_featurizer
+    kernel_sizes = [2, 3, 4]
+    settings = {
+        'n_embed_featurizer': 768,
+        'n_layer': 1,
+        'num_layers_trained': 1,
+        'kernel_sizes': kernel_sizes,
+        'num_filters_per_size': 2,
```

Comment:
This is one where I could see sensical value for this model being anywhere from 1 - 100.  Definitely worth looking into, like that you started w/ a small value though.